content/applications/inventory_and_mrp/inventory/shipping/setup/sendcloud_shipping.rst

```diff
@@ -0,0 +1,187 @@
+
+To get started, go to :guilabel:`Odoo's Apps Module` and search for the module:
+:guilabel:`Sendcloud Shipping`. Install this module, if it isn't already installed.
+
+.. image:: sendcloud_shipping/sendcloud-mod.png
+   :align: center
+   :alt: Sendcloud Module in the Odoo App's Modules.
+
+Sendcloud Shipping Connector Configuration
+------------------------------------------
+
+Ensure the :guilabel:`Sendcloud Shipping Module` is activated in :menuselection:`Inventory -->
+Configuration --> Settings`. The :guilabel:`Sendcloud Shipping Connector` is found under the
+:guilabel:`Shipping Connectors` section.
+
+After activating the :guilabel:`Sendcloud Shipping Connector`, click on the link below the listed
+connector that reads: :guilabel:`Sendcloud Shipping Methods`. Once on the :guilabel:`Shipping
+Methods` page, click :guilabel:`Create`.
+
+.. tip::
+   :guilabel:`Shipping Methods` can also be accessed by going to the :guilabel:`Configuration menu`
+   in the Inventory App.
+
+Fill out the following fields in the :guilabel:`New Shipping Method Form`:
+
+- :guilabel:`Shipping Method` = Sendcloud DPD.
+- :guilabel:`Provider` = (Reveal the dropdown, and select :guilabel:`Sendcloud`).
+- :guilabel:`Delivery Product` = (Set the product that was configured for this shipping method or
+  create a new product).
+- Enter the :guilabel:`Sendcloud Public Key`.
+- Enter the :guilabel:`Sendcloud Secret Key`.
+- Manually :guilabel:`Save` the work:
+
+  - Click on the :guilabel:`Cloud` icon to the right of the breadcrumbs.
+
+- Click on :guilabel:`Load the Sendcloud Shipping Products`.
+- Select the shipping products the company would like to use for deliveries and returns.
+- Click :guilabel:`Select`.
+
+For Example:
+
+.. image:: sendcloud_shipping/sendcloud-example.png
+   :align: center
+   :alt: Example of shipping products configured in Odoo.
+
+.. tip::
+   Sendcloud doesn't provide test keys when a company tests the sending of a package in Odoo. This
+   means, if a package is created, the configured Sendcloud account will be charged, unless the
+   associated package is canceled within 24 hours of creation.
+
+   Odoo has built an extra layer of protection into test environments. If the shipping method is
+   used to create a label, then the labels are immediately canceled after the creation. This occurs
+   automatically.
+
+Generate Label with Sendcloud
+-----------------------------
+
+When creating a quotation, just add shipping, and a :guilabel:`Sendcloud shipping product`, then
+:guilabel:`Validate` the delivery. Automatically, shipping label documents will be generated in the
+chatter.
+
+Included should be the following:
+
+#. :guilabel:`Shipping label(s)` (Depending on the number of packages).
+#. :guilabel:`Return label(s)` (If the Sendcloud connector is configured for this).
+#. :guilabel:`Customs document(s)` (Should the destination country require them).
+
+Additionally, the :guilabel:`tracking number` will be available.
```

Comment:
```suggestion
Additionally, the :guilabel:`tracking number` is also available.
```

@tiku-odoo Can you double check if `tracking number` is all lowercase on the interface? If not, can you update the capitalization to match how `tracking number` appears on the interface?

Reply:
Removed GUI labels. @mivu-odoo. This is more of a general reference.